vulcan/lib/server/models/workflow.rb

```diff
@@ -161,8 +162,15 @@ def input_as_json(input)
       def lookup_operation_script
         return nil if script_name.nil?
 
-        script_path = File.join(Vulcan.instance.config(:workflows_folder), "scripts/#{script_name}.py")
-        if ::File.exists?(script_path)
+        script_path = nil
+
+        Find.find(File.join(Vulcan.instance.config(:workflows_folder), "scripts/")) do |path|
```

Comment:
This change allows non-Python scripts to be found. It also means we can organize the scripts into project-specific ones, if that is the long-term goal to support editing.